### PR TITLE
Update predict-taxi-trip-duration demo notebooks

### PR DESCRIPTION
**Pull request by thehomebrewnerd**

### CODE REVIEW

File: predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb [hunk 1]
```diff
@@ -229,7 +233,7 @@
     "<h2>Step 3: Create baseline features using DFS </h2>\n",
     "<p>Instead of manually creating features, such as month of <b>pickup_datetime</b>, we can let featuretools come up with them. </p>\n",
     "\n",
-    "<p>Within featuretools there is a standard format for representing data that is used to set up predictions and build features. A <b>EntitySet</b> stores information about entities (database table), variables (columns in database tables), relationships, and the data itself. </p>\n",
+    "<p>Within featuretools there is a standard format for representing data that is used to set up predictions and build features. A <b>EntitySet</b> stores information about dataframes (database table), columns in the dataframes, relationships, and the data itself. </p>\n",
```
Comment (tamargrey) [inline predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb:236]: The wording of how entity sets are explained feels a little off to me. How about 

> An EntitySet stores typing information about the dataframe in the EntitySet, information about individual columns within each dataframe, the relationship between dataframes, and the data itself.

File: predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb [hunk 2]
```diff
@@ -247,9 +251,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<p>We can then use the `entity_from_dataframe` method to add an Entity called <i>trips</i>. We want to track the `id`, the `time_index` and specify other types of variables we care about in this entity. </p>\n",
+    "We can then use the `add_dataframe` method to add a dataframe called <i>trips</i>. We want to track the `id`, the `time_index` and specify other column types we care about in this table.\n",
     "\n",
-    "<p>As a note: Featuretools will try to interpret the types of variables. We can override this interpretation by specifying the types. In this case, I wanted <b>passenger_count</b> to be a type of Ordinal, and <b>vendor_id</b> to be of type Categorical.</p>"
+    "<p>As a note: Featuretools will try to interpret the types of columns. We can override this interpretation by specifying the types. In this case, I wanted <b>passenger_count</b> to be a type of Ordinal, and <b>vendor_id</b> to be of type Categorical.</p>"
```
Comment (tamargrey) [inline predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb:256]: Worth mentioning Woodwork here and linking to the logical types guide? https://woodwork.alteryx.com/en/stable/guides/logical_types_and_semantic_tags.html

File: predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb (hunk 1 above)
Comment (thehomebrewnerd) [reply predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb:236]: @tamargrey  Yeah, that was a little clunky. Taking what you wrote and thinking about the definition, I came up with this alternative. The multiple use "dataframe" combined with "the data itself" didn't feel quite right to me. What do you think about this?

> An <b>EntitySet</b> stores one or more data tables (dataframes), typing information for the columns in each of the tables and the parent-child relationships between the tables in the EntitySet.

Comment (tamargrey) [reply predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb:236]: I like that a lot!

File: predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb (hunk 2 above)
Comment (thehomebrewnerd) [reply predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb]: Yeah, added a sentence with that link.

**Review by tamargrey — APPROVED** [review]
lgtm!

**Review by dvreed77 — APPROVED** [review]
Finally got my environment working. LGTM!

**Pull request merged**